2020-06-23 04:47:30,118 ERROR: Exception on /search [GET] [in c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\ajshe\microblog2\app\main\routes.py", line 42, in search
    return render_template('search.html', title =_('Search'), search = g.search_form.q.data, total=total, posts=posts, utotal=utotal, users = users, next_url=next_url, prev_url=prev_url, DPForm=DPForm, UDForm=UDForm)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\ajshe\microblog2\app\templates\search.html", line 64, in template
    {% endfor %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endfor'. Jinja was looking for the following tags: 'endif'. The innermost block that needs to be closed is 'if'.
2020-06-23 04:49:07,685 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 04:49:10,604 ERROR: Exception on /search [GET] [in c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\ajshe\microblog2\app\main\routes.py", line 42, in search
    return render_template('search.html', title =_('Search'), search = g.search_form.q.data, total=total, posts=posts, utotal=utotal, users = users, next_url=next_url, prev_url=prev_url, DPForm=DPForm, UDForm=UDForm)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\ajshe\microblog2\app\templates\search.html", line 61, in template
    {% endif %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endif'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'endfor' or 'else'. The innermost block that needs to be closed is 'for'.
2020-06-23 04:51:53,041 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 04:55:44,392 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 04:55:46,972 ERROR: Exception on /search [GET] [in c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\ajshe\microblog2\app\main\routes.py", line 42, in search
    return render_template('search.html', title =_('Search'), search = g.search_form.q.data, total=total, posts=posts, utotal=utotal, users = users, next_url=next_url, prev_url=prev_url, DPForm=DPForm, UDForm=UDForm)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "c:\users\ajshe\microblog2\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\ajshe\microblog2\app\templates\search.html", line 100, in template
    {% endblock %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'endif'. The innermost block that needs to be closed is 'if'.
2020-06-23 04:57:42,299 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 04:58:06,662 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 05:06:17,059 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 05:08:24,737 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 05:09:45,945 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
2020-06-23 05:10:43,795 INFO: Microblog Startup [in C:\Users\ajshe\microblog2\app\__init__.py:77]
